Prace_domowe/Praca_domowa5/Grupa2/Piotr_Marciniak/pd5.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>Myślę że wynika to w dużej mierze z faktu, że dla tych danych dla ustalonej liczby klastrów w różnych modelach utworzone zostaną podobne klastry. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>